README.md

```diff
@@ -26,6 +26,7 @@ Available Commands:
   node-density                   Runs node-density workload
   node-density-cni               Runs node-density-cni workload
   node-density-heavy             Runs node-density-heavy workload
+  node-density-udn               Runs node-density-udn workload
```

Comment:
Can we rename this to node-density-udn-l3 (and add a layer2 workload)

Reply:
Rename it or maintain the current name and have a variable for it?
I am good either way.

Reply:
Why not `udn-density`, to avoid confusions with node-density workloads

Reply:
i wonder if variables will become just a mess of vars -- might make sense to build more specific workloads vs trying to var everything, wdyt?

Reply:
Renamed

Reply:
cc @vishnuchalla